openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -28,29 +28,31 @@
 - control_key: AU-2
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-155
+        OpenShift has an API audit system which would need to be enabled
```

Comment:
We'll need to be a bit more specific about what OpenShift can audit out of the box. The most common questions revolve around the ability to meet requirements from :

- OSPP Configuration Annex
https://www.niap-ccevs.org/MMO/PP/-442ConfigAnnex-/

- ICD 500-27 (scroll to pdf pg7)
https://www.dni.gov/files/documents/FOIA/DF-2016-00213.pdf